### ISSUE TITLE
'TimeZoneField' has no attribute 'default_choices'

### ISSUE THREAD

**Issue by pv8** [mfogel/django-timezone-field#76]
After upgrading `django-timezone-field`  to v4.2, Django (v2.2.24) application is not able to initialize: 

```sh
    def crontab_schedule_celery_timezone():
        """Return timezone string from Django settings `CELERY_TIMEZONE` variable.

        If is not defined or is not a valid timezone, return `"UTC"` instead.
        """
        try:
            CELERY_TIMEZONE = getattr(
                settings, '%s_TIMEZONE' % current_app.namespace)
        except AttributeError:
            return 'UTC'
        return CELERY_TIMEZONE if CELERY_TIMEZONE in [
>           choice[0].zone for choice in timezone_field.
            TimeZoneField.default_choices
        ] else 'UTC'
E       AttributeError: type object 'TimeZoneField' has no attribute 'default_choices'

../.local/lib/python3.8/site-packages/django_celery_beat/models.py:70: AttributeError
```

**Comment by mfogel** [comment]
Hi @pv8, thanks for the bug report

I'll go ahead and reinstate that `.default_chocies` property, will release a bugfix release shortly. Stay tuned.

**Comment by pv8** [comment]
That was fast! Thanks, @mfogel 👍🏿

**Comment by mfogel** [comment]
No prob. Released as part of `4.2.1` just now, let me know if you run into any further issues.

**Comment by charleshan** [comment]
It looks like `default_choices` was removed again: https://github.com/mfogel/django-timezone-field/commit/93fc0110c7c92ff9531c1373d98107f250450bd1#diff-045103e563771678972ccf52a093454db0843a046fdde3054fdd0e878eda6559L56
